src/PageLayout/PageLayout.tsx

```diff
@@ -114,7 +114,6 @@ const HorizontalDivider: React.FC<DividerProps> = ({variant = 'none', variantWhe
   const {padding} = React.useContext(PageLayoutContext)
   return (
     <Box
-      role="separator"
```

Comment:
ooh interesting, what does removing this do?

Reply:
If doesn't have role= separator should this have aria-hidden=true instead?

Reply:
Good callout @siddharthkp. I believe my recommendation was to use both `role="presentation"` and `aria-hidden="true"`. @colebemis, I'll update the recommendation in the a11y review issue.

Reply:
Is this necessary to do on a `div` element?

Reply:
Without adding `role` or `aria-hidden`, it like looks divider already does not appear in the accessibility tree: 

![CleanShot 2022-03-17 at 16 25 07@2x](https://user-images.githubusercontent.com/4608155/158910051-6998180b-ba03-4fe2-8450-8a5c34ba72c8.png)

What do `role` and `aria-hidden` do for us in this case?

Reply:
Ooh because it's an empty div, it would probably just get "Ignored" in the tree